plugins/obs-ffmpeg/obs-ffmpeg-audio-encoders.c

```diff
@@ -265,7 +264,6 @@ static void *enc_create(obs_data_t *settings, obs_encoder_t *encoder,
 	}
 
 	if (strcmp(enc->codec->name, "aac") == 0) {
```

Comment:
Since the if clause is now empty it probably should've been removed entirely

Reply:
I reached the commit a89470d2ea1 left the empty `if` clause. Then, aa58b9cf5 reuse the empty clause and I revert aa58b9cf5. I think it's safe to remove the empty `if` clause.